Week1/homework/js2-exercises/4-theTime.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Klok :)</title>
+    <style>
+        div {
```

Comment:
For the same reason explained above I believe it is better to use a css file and keep style in there instead of internal style tag